corsheaders/checks.py

```diff
@@ -0,0 +1,97 @@
+import collections
+import numbers
+
+from django.core import checks
+from .conf import corsheaders_settings
+
+@checks.register
+def check_settings(app_configs, **kwargs):
+
+    errors = []
+    if not isinstance(corsheaders_settings.CORS_ALLOW_HEADERS, collections.Sequence):
+        errors.append(
+            checks.Error(
+                "CORS_ALLOW_HEADERS must be sequence",
+                id="corsheaders.W001"
```

Comment:
just realized - since we're now returning `Error`, these codes should start with E rather than W